docs/docs/tutorials/contract_tutorials/counter_contract.md

```diff
@@ -160,28 +160,4 @@ In the same directory, run this:
 aztec-builder codegen -o src/artifacts target
 ```
 
-You can now use the artifact and/or the TS class in your Aztec.js! If you skipped the Aztec.js getting-started guide, you can follow it [here](aztecjs-getting-started.md). This will teach you about deploying and calling contracts in Aztec.js.
-
-## Install Noir LSP (recommended)
```

Comment:
Should we add this to the getting started section? If not, where should we recommend people install this?